flows/engine/session.go

```diff
@@ -429,12 +450,6 @@ func ReadSession(cache *AssetCache, assetURLs map[AssetItemType]string, data jso
 		s.addRun(run)
 	}
 
-	// once all runs are read, we can resolve references between runs
-	err = runs.ResolveReferences(s, s.Runs())
```

Comment:
Now that we're not worried about children, this isn't necessary because a run's parent will always have been loaded before it - runs in the session are chronological.